astropy/table/tests/test_mixin.py

```diff
@@ -124,13 +124,13 @@ def test_votable_quantity_write(tmpdir):
     (io/fits/tests/test_connect.py and io/misc/tests/test_hdf5.py).
     """
     t = QTable()
-    t['a'] = u.Quantity([1, 2, 4], unit='Angstrom')
+    t['a'] = u.Quantity([1, 2, 4], unit='m')
```

Comment:
Does the new format not support Angstrom? My tendency would then be to use 'nm' to at least keep it a prefixed unit.

Reply:
Angstroms are supported but discouraged, so using them raises a UnitsWarning which seems to cause this test to fail. Running `pytest -P table -p no:warnings` does succeed.

Reply:
OK, let's use 'nm' instead, then.